### PR TITLE
add a quickstart tutorial

### LINKED ISSUE #29: Docs: simpler 1st example
The first example in docs (metabolic modeling) isn't very beginner-friendly.

Make up something much simpler, like the wiki example with factories. Or EMD.

### PR DESCRIPTION
**Pull request by exaexa**
I guess this should enable 1.0 getting out.

Closes #29

### CODE REVIEW

File: docs/src/0-quickstart.jl [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+# Let's name our system `s`. We first need a few variables:
+
+s = C.variables(keys = [:wheat, :barley])
+
+# With ConstraintTrees.jl, we can (and want to!) label everything very nicely
+# -- the constraint trees are essentially directory structures, so one can
+# prefix everything with symbols to put it into nice directories.
+
+s = :area^s
```
Comment (stelmo) [inline docs/src/0-quickstart.jl:57]: this might confuse some, why is `s` being overwritten here?

File: docs/src/0-quickstart.jl [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            JuMP.@constraint(model, C.substitute(c.value, x) == b.equal_to)
+        elseif b isa C.Between
+            val = C.substitute(c.value, x)
+            isinf(b.lower) || JuMP.@constraint(model, val >= b.lower)
+            isinf(b.upper) || JuMP.@constraint(model, val <= b.upper)
+        end
+    end
+    JuMP.optimize!(model)
+    JuMP.value.(model[:x])
+end
```
Comment (stelmo) [inline docs/src/0-quickstart.jl:127]: It would be more convenient if this was built in tbh

**Review by stelmo — APPROVED** [review]
This is great, but maybe the JuMP model conversion can be built in?

File: docs/src/0-quickstart.jl (hunk 1 above)
Comment (exaexa) [reply docs/src/0-quickstart.jl:57]: true

**Review by stelmo — APPROVED** [review]

**Pull request merged**